ui/components/multichain/asset-picker-amount/asset-picker/asset-picker.tsx

```diff
@@ -41,13 +41,15 @@ export type AssetPickerProps = {
    * Sending asset for UI treatments; only for dest component
    */
   sendingAsset?: Asset;
+  isDisabled?: boolean;
 };
 
 // A component that lets the user pick from a list of assets.
 export function AssetPicker({
   asset,
   onAssetChange,
   sendingAsset,
+  isDisabled = false,
```

Comment:
Should we outright disable it? The current functionality shows the "send" option for both when "disabled"